WordPress/Classes/Services/BlogService.m

```diff
@@ -443,12 +444,23 @@ - (Blog *)findBlogWithXmlrpc:(NSString *)xmlrpc
 
 - (Blog *)createBlogWithAccount:(WPAccount *)account
 {
-    Blog *blog = [NSEntityDescription insertNewObjectForEntityForName:NSStringFromClass([Blog class])
+    NSString *entityName = NSStringFromClass([Blog class]);
+    Blog *blog = [NSEntityDescription insertNewObjectForEntityForName:entityName
                                                inManagedObjectContext:self.managedObjectContext];
     blog.account = account;
+    blog.settings = [self createSettingsWithBlog:blog];
     return blog;
 }
 
+- (BlogSettings *)createSettingsWithBlog:(Blog *)blog
+{
+    NSString *entityName = [BlogSettings classNameWithoutNamespaces];
```

Comment:
The things you will do to avoid typing a string directly... :trollface: 
Now seriously, if this is needed perhaps it's a sign that we shouldn't be assuming that class name == entity name, and type it directly. If you can't let it go and just type `@"BlogSettings"` in there, I'd recommend adding a `static let entityName = "BlogSettings"` in `BlogSettings` and use that